ras/packages.py

```diff
@@ -24,6 +24,10 @@
 class Package_check(Test):
 
     def setUp(self):
+
```

Comment:
Minor nit, can the extra line be removed ?

Reply:
@kamalesh-babulal sure will do